activerecord/lib/active_record/connection_adapters/abstract/schema_statements.rb

```diff
@@ -317,6 +317,8 @@ def create_table(table_name, comment: nil, **options)
       # [<tt>:force</tt>]
       #   Set to true to drop the table before creating it.
       #   Defaults to false.
+      # [<tt>:type</tt>]
+      #   The column type of primary key. Defaults to +:integer+.
```

Comment:
@y-yagi I merged it fast. But the `type` is coming from `column_options` not directly.

``` ruby
type = column_options.delete(:type) || :integer
```

right?